ra_aid/tools/programmer.py

```
@@ -76,6 +83,7 @@ def run_programming_task(instructions: str, files: List[str] = []) -> Dict[str,
 
     markdown_content = "".join(task_display)
     console.print(Panel(Markdown(markdown_content), title="🤖 Aider Task", border_style="bright_blue"))
+    logger.debug(f"command: {command}")
```

Ruff (0.8.2)

Logging statement uses f-string

(G004)